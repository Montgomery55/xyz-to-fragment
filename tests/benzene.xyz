12

  H    1.2193786   -0.1652071    2.1600143
  C    0.6824786   -0.0924071    1.2087143
  C   -0.7075214   -0.0352071    1.1973143
  H   -1.2644214   -0.0630071    2.1393143
  C   -1.3898214    0.0571929   -0.0113857
  H   -2.4836214    0.1020929   -0.0203857
  C   -0.6824214    0.0924929   -1.2087857
  H   -1.2194214    0.1651929   -2.1598857
  C    0.7074786    0.0351929   -1.1972857
  H    1.2640786    0.0627929   -2.1394857
  C    1.3898786   -0.0572071    0.0114143
  H    2.4835786   -0.1022071    0.0205143
