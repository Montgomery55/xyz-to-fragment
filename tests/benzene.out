12
1    H          1.21937860  -0.16520710   2.16001430	11    	2
2    C          0.68247860  -0.09240710   1.20871430	63    	1 3 11
3    C         -0.70752140  -0.03520710   1.19731430	63    	2 4 5
4    H         -1.26442140  -0.06300710   2.13931430	11    	3
5    C         -1.38982140   0.05719290  -0.01138570	63    	3 6 7
6    H         -2.48362140   0.10209290  -0.02038570	11    	5
7    C         -0.68242140   0.09249290  -1.20878570	63    	5 8 9
8    H         -1.21942140   0.16519290  -2.15988570	11    	7
9    C          0.70747860   0.03519290  -1.19728570	63    	7 10 11
10   H          1.26407860   0.06279290  -2.13948570	11    	9
11   C          1.38987860  -0.05720710   0.01141430	63    	2 9 12
12   H          2.48357860  -0.10220710   0.02051430	11    	11
